sphinx_gallery/gen_gallery.py

```diff
@@ -527,9 +528,10 @@ def get_subsections(srcdir, examples_dir, gallery_conf, check_for_header=True):
     if gallery_conf["subsection_order"] is None:
         sortkey = None
     else:
+        subsec_order = gallery_conf.get("subsection_order")
+        if isinstance(subsec_order, list):
+            gallery_conf["subsection_order"] = ExplicitOrder(subsec_order)
```

Comment:
this is the change that makes bare lists work for `subsection_order` (must be wrapped in `ExplicitOrder` before the call to `_get_callables`)